harness-evals/harness-docs/error-handling-guidelines.md

```
@@ -0,0 +1,167 @@
+# Error Handling Guidelines
+
+Scope: the controller-runtime (ctrl-runtime) stack — **IstioCSR** and **TrustManager**. CertManager
+uses library-go and follows a different model (see [§7](#7-library-go-vs-ctrl-runtime-do-not-mix-patterns)).
```

markdownlint-cli2 (0.23.1)

[warning] Link fragments should be valid

(MD051, link-fragments)